spec/models/course_spec.rb

```diff
@@ -434,4 +434,110 @@
       its(:content_status) { should eq(:published_with_unpublished_changes) }
     end
   end
+
+  describe "#publish_enrichments" do
+    context 'on a course with only a draft enrichment' do
+      let(:course) do
+        create(:course,
+            changed_at: 10.minutes.ago,
+          with_enrichments: [
```

Comment:
The indentation doesn't look quite right here

Reply:
rearranged, thanks